src/account/PluginManagerInternals.sol

```diff
@@ -103,19 +102,28 @@ abstract contract PluginManagerInternals is IPluginManager {
         SelectorData storage _selectorData = getAccountStorage().selectorData[selector];
 
         if (!preExecHook.isEmpty()) {
-            _addOrIncrement(_selectorData.preHooks, _toSetValue(preExecHook));
+            if (preExecHook.eq(FunctionReferenceLib._PRE_HOOK_ALWAYS_DENY)) {
+                // Increment the overlappingDenies counter.
```

Comment:
Nit: update comment - is this a reference to an earlier version of the variable name?

Reply:
Yes it was, updating